filters/auth/authclient.go

```diff
@@ -74,6 +76,7 @@ func (ac *authClient) doClonedGet(ctx filters.FilterContext) (int, error) {
 	parentSpan := ctx.ParentSpan()
 	request := ctx.Request()
 	span := tracer.StartSpan(webhookSpanName, opentracing.ChildOf(parentSpan.Context()))
+	span.SetTag("target", ac.url.String())
```

Comment:
I believe there are some [established conventions](https://github.com/opentracing/specification/blob/master/semantic_conventions.md) for this.
Either `http.url` or `peer.service` seem like good candidates.